exporter/loggingexporter/known_sync_error.go

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:build !windows
-// +build !windows
+//go:build unix
+// +build unix
```

Comment:
:facepalm: sorry, the `unix` build tag was added on Go 1.19 so we can't use this yet since we support Go 1.18.

I believe something like this should make the build happy until we drop support for Go 1.18
```suggestion
//go:build linux || darwin
// +build linux darwin
```

Reply:
:relaxed: 
question: It is better to merge this suggestion? or is better to fix my local repo and push force?

Reply:
Either is fine, whatever is more comfortable for you :)